File: TechReady/07-16/queue.rb
require "azure"


Azure.config.sb_namespace = "nsgtrnamespace"
Azure.config.sb_access_key = "KbuVe06trwIaVPeYYg/F+WfmojIi+pdWe4SrD5BBczY="

message = Azure::ServiceBus::BrokeredMessage.new("test queue message")
message.correlation_id = "test-correlation-id"
azure_service_bus_service.send_queue_message("test-queue", message)

message = azure_service_bus_service.receive_queue_message("test-queue",
                                                          { :peek_lock => false })

print message

File: TechReady/07-16/queue.py
from azure.servicebus import ServiceBusService, Message, Queue

namespace = "nsgtrnamespace"
endpoint = "sb://" + namespace + ".servicebus.windows.net/"
keyName = "all"
key = "KbuVe06trwIaVPeYYg/F+WfmojIi+pdWe4SrD5BBczY="
queueName = "nsgtrqueue"

bus_service = ServiceBusService(
    service_namespace=namespace,
    shared_access_key_name=keyName,
    shared_access_key_value=key
)

sentMsg = Message(b'hi')
bus_service.send_queue_message(queueName, sentMsg)
receivedMsg = bus_service.receive_queue_message(queueName, peek_lock=False)
print(receivedMsg.body)

#print(bus_service.receive_queue_message(queueName, peek_lock=False))
